llama_index/agent/openai_agent.py

```diff
@@ -337,10 +337,15 @@ def chat(
         chat_history: Optional[List[ChatMessage]] = None,
         function_call: Union[str, dict] = "auto",
     ) -> AgentChatResponse:
-        chat_response = self._chat(
-            message, chat_history, function_call, mode=ChatResponseMode.WAIT
-        )
-        assert isinstance(chat_response, AgentChatResponse)
+        with self.callback_manager.event(
+            CBEventType.AGENT_STEP,
+            payload={EventPayload.MESSAGES: [message]},
```

Comment:
hey @logan-markewich - This is the first time I'm seeing a plain string be passed in the messages list. Was this intentional to not use the `ChatMessage` class?

Reply:
Hmm, yea my bad I think. I can patch this tomorrow